### PR TITLE
Don't run an e2e test if recording the example failed (#6467)

### PR DESCRIPTION
**Pull request by hbenl**

### CODE REVIEW

**Review by jasonLaster — APPROVED** [review]
Good call

**Comment by jasonLaster** [comment]
@hbenl could you share a description. I assume this fixes some cases we were seeing, but would be nice to have the context for historical records

**Comment by hbenl** [comment]
> I assume this fixes some cases we were seeing, but would be nice to have the context for historical records

Sorry, I had forgotten to link the issue.

File: test/run.js [hunk 1]
```diff
@@ -50,9 +50,14 @@ async function runMatchingTests(state) {
...
       const exampleRecordingId = await getExample(state, example, target);
-      console.log(
-        `[${elapsedTime(state)}] TestExample:${test} target:${target} ${exampleRecordingId}`
-      );
+      if (exampleRecordingId) {
+        console.log(
+          `[${elapsedTime(state)}] TestExample:${test} target:${target} ${exampleRecordingId}`
+        );
+      } else {
+        console.log(`::error ::Failure Failed to record example ${example}`);
```
Comment (bvaughn) [inline test/run.js:58]: Is there any structure to our logging format here? Curious why "TestStart" and "TestExample" vs "::error ::"
```js
console.log(`[${elapsedTime(state)}] TestStart:${test} target:${target}`);
console.log(`[${elapsedTime(state)}] TestExample:${test} target:${target} ${exampleRecordingId}`);
console.log(`::error ::Failure Failed to record example ${example}`);
```

**Pull request merged**